<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [rabbitmq-discuss] RabbitMQ 3.2.2 durable queue failover
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:rabbitmq-discuss%40lists.rabbitmq.com?Subject=Re%3A%20%5Brabbitmq-discuss%5D%20RabbitMQ%203.2.2%20durable%20queue%20failover&In-Reply-To=%3C933321ee-bfc7-4ee9-9265-aac2d79d2c11%40googlegroups.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="034204.html">
   <LINK REL="Next"  HREF="034098.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[rabbitmq-discuss] RabbitMQ 3.2.2 durable queue failover</H1>
    <B>Andy Martin</B> 
    <A HREF="mailto:rabbitmq-discuss%40lists.rabbitmq.com?Subject=Re%3A%20%5Brabbitmq-discuss%5D%20RabbitMQ%203.2.2%20durable%20queue%20failover&In-Reply-To=%3C933321ee-bfc7-4ee9-9265-aac2d79d2c11%40googlegroups.com%3E"
       TITLE="[rabbitmq-discuss] RabbitMQ 3.2.2 durable queue failover">akmartin at gmail.com
       </A><BR>
    <I>Mon Feb 24 19:57:39 GMT 2014</I>
    <P><UL>
        <LI>Previous message: <A HREF="034204.html">[rabbitmq-discuss] RabbitMQ Conference?
</A></li>
        <LI>Next message: <A HREF="034098.html">[rabbitmq-discuss] RabbitMQ 3.2.2 durable queue failover
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#34077">[ date ]</a>
              <a href="thread.html#34077">[ thread ]</a>
              <a href="subject.html#34077">[ subject ]</a>
              <a href="author.html#34077">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>I am testing failover before putting my RabbitMQ implementation into 
production. When I bring down one RabbitMQ node I start getting the 
following error:

RabbitMQ.Client.Exceptions.OperationInterruptedException: The AMQP 
operation was interrupted: AMQP close-reason, initiated by Peer, code=404, 
text=&quot;NOT_FOUND - home node '<A HREF="https://lists.rabbitmq.com/cgi-bin/mailman/listinfo/rabbitmq-discuss">rabbit at DEVRABBIT01</A>' of durable queue 
'QUEUE_TEST' in vhost '/' is down or inaccessible&quot;, classId=50, 
methodId=10, cause=

I am running RabbitMQ 3.2.2 with Erlang R16B03.

DEVRABBIT01 is the node that has been shut down. 
DEVRABBIT02 is the node that is still running and that the client is 
connected to.

Both are disc nodes. The named queue (QUEUE_TEST) does not appear in the 
management console on the running node.

It looks like this was supposed to have been fixed in 2.4.0. Any reason I 
would be seeing this in 3.2.2?
-------------- next part --------------
An HTML attachment was scrubbed...
URL: &lt;<A HREF="http://lists.rabbitmq.com/pipermail/rabbitmq-discuss/attachments/20140224/5bf30e5c/attachment.html">http://lists.rabbitmq.com/pipermail/rabbitmq-discuss/attachments/20140224/5bf30e5c/attachment.html</A>&gt;
</PRE>










<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="034204.html">[rabbitmq-discuss] RabbitMQ Conference?
</A></li>
	<LI>Next message: <A HREF="034098.html">[rabbitmq-discuss] RabbitMQ 3.2.2 durable queue failover
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#34077">[ date ]</a>
              <a href="thread.html#34077">[ thread ]</a>
              <a href="subject.html#34077">[ subject ]</a>
              <a href="author.html#34077">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.rabbitmq.com/cgi-bin/mailman/listinfo/rabbitmq-discuss">More information about the rabbitmq-discuss
mailing list</a><br>
</body></html>
